python_modules/dagster/dagster/_core/storage/alembic/versions/035_add_mutable_partitions_table.py

```diff
@@ -0,0 +1,49 @@
+"""add runtime partitions table
+
+Revision ID: e62c379ac8f4
+Revises: 6df03f4b1efb
+Create Date: 2023-01-19 11:41:41.062228
+
+"""
+import sqlalchemy as db
+from alembic import op
+from dagster._core.storage.migration.utils import has_index, has_table
+from dagster._core.storage.sql import get_current_timestamp
+from sqlalchemy.dialects import sqlite
+
+# revision identifiers, used by Alembic.
+revision = "e62c379ac8f4"
+down_revision = "6df03f4b1efb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if not has_table("mutable_partitions_definitions"):
+        op.create_table(
+            "mutable_partitions_definitions",
```

Comment:
It doesn't quite roll of the tongue, but my instinct would be to name this something like `mutable_partitions_definitions_partitions`, because each element represents a (partitions_definition, partition_key) tuple, not just a partitions_definition. Would defer to @prha.

Reply:
`mutable_partitions_definitions_partitions` seems like too much... I prefer something more terse

Reply:
Let's go with `mutable_partitions`.